src/Compilers/CSharp/Test/Symbol/Symbols/DefaultInterfaceImplementationTests.cs

```diff
@@ -6,6 +6,7 @@
 
 using System;
 using System.Linq;
+using Basic.Reference.Assemblies;
```

Comment:
Is this using necessary?

Reply:
Yes, the use of `Net70.SystemRuntime` requires it